uvloop/loop.pyx

```diff
@@ -1074,8 +1080,10 @@ cdef class Loop:
             raise
         except BaseException as exc:
             fut.set_exception(exc)
+            self._remove_writer(sock)
```

Comment:
Hm, I'm not sure how did it work before? I.e. who removed the writer?

Reply:
> ```python
> fut.add_done_callback(lambda fut: self._remove_writer(sock))
> ```
This one I believe

Reply:
Had to remind myself what _SyncSocketWriterFuture is...